src/main/java/com/spring/onlinebookstore/annotation/FieldMatchValidator.java

```diff
@@ -0,0 +1,23 @@
+package com.spring.onlinebookstore.annotation;
+
+import com.spring.onlinebookstore.dto.user.UserRegistrationRequestDto;
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class FieldMatchValidator implements
+        ConstraintValidator<FieldMatch, UserRegistrationRequestDto> {
+
+    @Override
+    public void initialize(FieldMatch constraintAnnotation) {
+    }
+
+    @Override
+    public boolean isValid(UserRegistrationRequestDto userRegistrationRequestDto,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        if (userRegistrationRequestDto == null) {
```

Comment:
did you check this validation for other cases? when password and repeatPassword null, when password and repeatPassword not equals?